google-cloud-storage/src/test/scala/akka/stream/alpakka/googlecloud/storage/scaladsl/GCStorageWiremockBase.scala

```diff
@@ -76,7 +76,10 @@ abstract class GCStorageWiremockBase(_system: ActorSystem, _wireMockServer: Hove
        |  "customerEncryption": {"encryptionAlgorithm": "AES256", "keySha256": "encryption-key-sha256"},
        |  "owner": {"entity": "project-owners-123412341234", "entityId": "790607247"},
        |  "acl": [{ "kind": "storage#objectAccessControl", "id": "my-bucket/test-acl/1463505795940000", "selfLink": "https://www.googleapis.com/storage/v1/b/my-bucket/o/test-acl", "bucket": "my-bucket", "object": "test-acl", "generation": "1463505795940000", "entity": "some-entity", "role": "OWNER", "email": "owner@google.com", "entityId": "790607247", "domain": "my-domain", "projectTeam": { "projectNumber": "57959400", "team": "management" }, "etag": "R2OZrfQiij=" }]
-       |}""".stripMargin
+       |  ${maybeMd5Hash.fold("")(hash => s""","md5Hash":"$hash"""")}
```

Comment:
```suggestion
       |  ${maybeMd5Hash.fold("")(hash => s""","md5Hash":"$hash"""")},
```

Also missing comma at end of L78

Reply:
The comma is not needed. I have added it at the beginning of the string interpolation when the optional parameters are defined.